kuma/javascript/ssr-server.js

```diff
@@ -13,6 +13,21 @@ if (process.env.NEW_RELIC_LICENSE_KEY && process.env.NEW_RELIC_APP_NAME) {
     require('newrelic');
 }
 
+if (process.env.SENTRY_DSN) {
+    console.log('Configuring Sentry for the SSR server.');
+    const Sentry = require('@sentry/node');
```

Comment:
I don't know how I like this. I'd rather this was move outside the if statement so that we know for sure that the `require` works even if you don't configure and call `.init(options)` on it.

Reply:
Actually, what I'd like to is move ALL of this code *except* the line `const Sentry = require('@sentry/node');` to somewhere further down after all the `require()` stuff has happened.

Reply:
I'll do that.